src/pages/edge/adobe-journey-optimizer/proposition-edge-events.md

```diff
@@ -0,0 +1,354 @@
+---
+title: Adobe Journey Optimizer proposition edge events
+description: Outlines proposition-related edge events sent by Adobe Journey Optimizer (AJO) mobile extension.
+keywords:
+- Adobe Journey Optimizer
+- Messaging
+- Product overview
+- Edge events
+---
+
+# AJO edge events for propositions
+
+This page outlines each event type generated by the AEPMessaging extension related to propositions, which includes in-app messages, content cards, and code-based experiences.
+
+The following XDM event types are used in proposition tracking:
+
+| Event Type | Description |
+|----|----|
+| `decisioning.propositionDismiss` | The proposition that was being displayed to the user has been removed from the user interface. |
+| `decisioning.propositionDisplay` | The proposition has been displayed to the user. |
+| `decisioning.propositionDisqualify` | The user has been permanently disqualified from seeing the proposition. |
```

Comment:
this event type is not yet available on the android messaging side. should we add a note to mention that it is iOS only / will be available soon on android?